packages/safe-tools-client/app/components/unsupported-network-modal/index.gts

```diff
@@ -0,0 +1,61 @@
+import Component from '@glimmer/component';
+import { inject as service} from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { on } from '@ember/modifier';
+import arrayJoin from '@cardstack/safe-tools-client/helpers/array-join';
+import NetworkService from '@cardstack/safe-tools-client/services/network';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+import BoxelActionContainer from '@cardstack/boxel/components/boxel/action-container';
+import BoxelModal from '@cardstack/boxel/components/boxel/modal';
+
+import './index.css';
+
+interface Signature {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+class UnsupportedNetworkModal extends Component<Signature> {
+  @service declare network: NetworkService;
+
+  @tracked chosenProviderId: string | undefined;
+
+  <template>
+    <BoxelModal
+      class='unsupported-network-modal'
+      @size='small'
+      @isOpen={{@isOpen}}
+      @onClose={{@onClose}}
+      data-test-unsupported-network-modal
+    >
+    <BoxelActionContainer
+          class='unsupported-network-modal__card'
+          tabindex='-1'
+          as |Section|
+        >
+          <button
+            class='unsupported-network-modal__close-button'
```

Comment:
Have you considered making this modal the same way as "Authenticate with the Cardstack Hub" is made? You can add the close button in the `ActionChin`. That way you can get rid of most if not all of the custom CSS you have added.

This is a very basic modal so let's try to reuse the generic solutions that we have